draft-ietf-lamps-kyber-certificates.md

```diff
@@ -295,9 +320,37 @@ encoded using the textual encoding defined in {{?RFC7468}}.
 
 # Security Considerations
 
-The Security Considerations section of {{RFC5280}} applies to this specification as well.
-
-For ML-KEM specific security considerations refer to {{?I-D.sfluhrer-cfrg-ml-kem-security-considerations}}.
+The Security Considerations section of {{RFC5280}} applies to this
+specification as well.
+
+Protection of the private-key information, i.e., the seed, is vital to public-key cryptography.  Disclosure of the private-key material to another entity can lead to masquerades.
+
+For ML-KEM specific security considerations refer to
+{{?I-D.sfluhrer-cfrg-ml-kem-security-considerations}}.
+ 
+ The generation of private keys relies on random numbers. The use of
+ inadequate pseudo-random number generators (PRNGs) to generate these
+ values can result in little or no security.  An attacker may find it
+ much easier to reproduce the PRNG environment that produced the keys,
+ searching the resulting small set of possibilities, rather than brute
+ force searching the whole key space.  The generation of quality
+ random numbers is difficult, and {{?RFC4086}} offers important guidance
+ in this area.
+ 
```

Comment:
```suggestion
The generation of private keys relies on random numbers. The use of
inadequate pseudo-random number generators (PRNGs) to generate these
values can result in little or no security.  An attacker may find it
much easier to reproduce the PRNG environment that produced the keys,
searching the resulting small set of possibilities, rather than brute
force searching the whole key space.  The generation of quality
random numbers is difficult, and {{?RFC4086}} offers important guidance
in this area.
 
```